tech/backups.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Backing up FAIRDOM-SEEK
+title: Backup and Restore
```

Comment:
```suggestion
title: Backup and restore
```
Also, this new title has not been reflected in the `_data/sidebars/tech.yml` side menu file.